api_management/apps/analytics/csv_generator.py

```diff
@@ -35,12 +94,78 @@ def all_queries(self):
                                     start_time__gte=min_date,
                                     start_time__lt=max_date).exclude(request_method='OPTIONS')
 
-    def create_csv_file(self, file_name, file):
-        csv_file = CsvFile.objects.filter(api_name=self.api_name, file_name=file_name).first()
-        if csv_file is not None:
-            if csv_file.file.name != '':
-                csv_file.file.delete()  # removes from disk
-            csv_file.file = File(file)
-            csv_file.save()
+
+def is_mobile(user_agent):
+    return "Mobile" in user_agent or \
+           "Android" in user_agent or \
+           "iPhone" in user_agent or \
+           "Slackbot" in user_agent
```

Comment:
`return user_agent in USER_AGENTS`, con `USER_AGENTS = ['Mobile', 'Android', 'iPhone', 'Slackbot']`

Reply:
`return any([keyword in user_agent for keyword in USER_AGENTS])`